README.md

```diff
@@ -172,18 +172,19 @@ USAGE:
     simple [FLAGS] [OPTIONS]
 
 FLAGS:
-    -h, --help             Prints help information
-    -l, --list             Shows list of all possible Goose tasks and exits
-    -g, --log-level        Log level (-g, -gg, -ggg, etc.)
-        --manager          Enables manager mode
-        --no-hash-check    Ignore worker load test checksum
-        --no-stats         Don't print stats in the console
-        --only-summary     Only prints summary stats
-        --reset-stats      Resets statistics once hatching has been completed
-        --status-codes     Includes status code counts in console stats
-    -V, --version          Prints version information
-    -v, --verbose          Debug level (-v, -vv, -vvv, etc.)
-        --worker           Enables worker mode
+    -h, --help                Prints help information
+    -l, --list                Shows list of all possible Goose tasks and exits
+    -g, --log-level           Log level (-g, -gg, -ggg, etc.)
+        --manager             Enables manager mode
+        --no-hash-check       Ignore worker load test checksum
+        --no-stats            Don't print stats in the console
+        --no-sticky-follow    Prevent load test client from following redirect of base_url with subsequent requests
```

Comment:
@nnewton @LionsAd Does this help-text make sense? It's hard to explain in so few words (though ideally the help text would be even shorter)

Reply:
I like the concept of "sticky follow" Makes sense to me.

Reply:
I like it, too